src/analysis/special_functions/gaussian.lean

```diff
@@ -0,0 +1,176 @@
+/-
+Copyright (c) 2022 Sébastien Gouëzel. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Sébastien Gouëzel
+-/
+import analysis.special_functions.gamma
+import analysis.special_functions.polar_coord
+
+/-!
+# Gaussian integral
+
+We prove the formula `∫ x, exp (-b * x^2) = sqrt (π / b)`, in `integral_gaussian`.
+-/
+
+noncomputable theory
+
+open real set measure_theory filter asymptotics
+open_locale real topological_space
+
+lemma exp_neg_mul_sq_is_o_exp_neg {b : ℝ} (hb : 0 < b) :
+  (λ x:ℝ, exp (-b * x^2)) =o[at_top] (λ x:ℝ, exp (-x)) :=
+begin
+  refine is_o_of_tendsto (λ x hx, _) _,
+  { exfalso, exact (exp_pos (-x)).ne' hx },
+  have : (λ (x:ℝ), exp (-b * x^2) / exp (-x)) = (λ (x:ℝ), exp (x * (1 - b * x))),
+  { ext1 x, field_simp [exp_ne_zero, real.exp_neg, ← real.exp_add], ring_exp },
+  rw this,
```

Comment:
I feel like there is some lemma to extract here, like `exp ∘ f =o exp ∘ g` iff `f - g` tendsto -∞

Reply:
Indeed, and this lemma has been added recently by @urkud in #15506 so I don't even need to prove it ;-)